### DISCUSSION TITLE
请问ClassIsland-net6仓库同步ClassIsland仓库的策略是如何的呢？

### DISCUSSION THREAD

**Discussion by mcAmiya** [ClassIsland/ClassIsland#549] [常规]
如题，请问ClassIsland-net6仓库同步ClassIsland仓库的策略是如何的呢，是要手动同步的吗，还是每次大版本手动同步一次呢，以及是否接受ClassIsland-net6仓库是否接受pr，还是说pr申请到ClassIsland仓库，然后ClassIsland-net6仓库等同步就行呢。望回复，十分感谢！

**Comment by HelloWRC** [comment]
ClassIsland-net6 仓库一般是在 master 分支上的版本发布时进行手动同步。PR 到主仓库的更改也会在同步时一起带入到 ClassIsland-net6 仓库中。

**Reply by mcAmiya** [reply]
感谢回复！由于班里是Win7 32位所以特意来咨询一下同步策略，看什么时候应该自行编译❤
